test/lib/compile/csdl2openapi.test.js

```diff
@@ -490,28 +490,42 @@ describe('Edge cases', function () {
         };
         const openapi = lib.csdl2openapi(csdl, {});
         assert.deepStrictEqual(
-            openapi.components.schemas["jsonExamples.typeDefinitionOld"],
-            {
-                title: "typeDefinitionOld",
+          openapi.components.schemas["jsonExamples.typeDefinitionOld"],
+          {
+            title: "typeDefinitionOld",
+            type: "object",
```

Comment:
why are we adding wrapper object for `type: "object"`?

Reply:
corrected